### PR TITLE
Jp Manage: Preview Pane - Layout with 4 placeholders: header, tabs bar, content, footer

### PR DESCRIPTION
**Pull request by gogdzl**
Related to https://github.com/Automattic/jetpack-manage/issues/292 and https://github.com/Automattic/jetpack-manage/issues/294

## Proposed Changes

* This PR adds a new preview panel with 4 placeholders,  header, tabs bar, content, footer, according to  https://github.com/Automattic/jetpack-manage/issues/292

## Testing Instructions

- Open the new sites listing ( http://jetpack.cloud.localhost:3000/sites )
- Click on any site
- The new preview panel should open
- Check the header according to https://github.com/Automattic/jetpack-manage/issues/294
  - The `X` (close button) should be functional
  - Clicking over the URL in the preview pane should open the website in a new window
  - The website's favicon is not yet implemented, it is using a placeholder for the moment.

*

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	closeSitePreviewPane: () => void;
+}
+
+export default function SitePreviewPaneHeader( { selectedSite, closeSitePreviewPane }: Props ) {
+	if ( ! selectedSite ) {
+		return null;
+	}
+
+	// eslint-disable-next-line no-console
+	console.log( selectedSite );
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx]: Don't forget to remove this console.log before merging this PR; it has the clause `eslint` disabled. 

Or better, remove it now.

Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx]: Removed in [2e167c1](https://github.com/Automattic/wp-calypso/pull/87760/commits/2e167c165473eec9400082d54b7c99dd51c64051)
Thanks!

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Site } from '../types';
+import SitePreviewPaneContent from './site-preview-pane-content';
+import SitePreviewPaneFooter from './site-preview-pane-footer';
+import SitePreviewPaneHeader from './site-preview-pane-header';
+import SitePreviewPaneTabs from './site-preview-pane-tabs';
+
+import './style.scss';
+
+interface Props {
+	selectedSite?: Site | undefined;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx]: The site should be a mandatory prop. So `?` and `undefined` wouldn't be necessary.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function SitePreviewPane( { selectedSite, closeSitePreviewPane }: Props ) {
+	if ( ! selectedSite ) {
+		return null;
+	}
+
+	return (
+		<div className="site-preview__pane">
+			<SitePreviewPaneHeader
+				selectedSite={ selectedSite }
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx]: We could send to the header component just the 3 variables it needs via props:

- title
- url?
- url_with_scheme?
- icon?
- closePreviewPane?

That way, we make it totally independent.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import { Icon, external } from '@wordpress/icons';
+import { translate } from 'i18n-calypso';
+import { Site } from '../../types';
+
+import './style.scss';
+
+const ICON_SIZE = 24;
+
+interface Props {
+	selectedSite?: Site | undefined;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx]: See my comment above.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+				margin-bottom: 4px;
+			}
+
+			.site-preview__header-summary {
+
+				.site-preview__header-summary-link {
+					display: flex;
+					align-items: center;
+					font-family: "SF Pro Text", sans-serif;
+					font-size: 18px; /* stylelint-disable-line declaration-property-unit-allowed-list */
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: Use `rem`

```suggestion
					font-size: rem(18px);
```

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+		.site-preview__header-title-summary {
+			flex-grow: 1;
+			display: flex;
+			flex-direction: column;
+
+			.site-preview__header-title {
+				font-family: "SF Pro Text", sans-serif;
+				font-style: normal;
+				font-weight: 500;
+				font-size: 32px; /* stylelint-disable-line declaration-property-unit-allowed-list */
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: Use `rem` (see below)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+				color: #000;
+				margin-bottom: 4px;
+			}
+
+			.site-preview__header-summary {
+
+				.site-preview__header-summary-link {
+					display: flex;
+					align-items: center;
+					font-family: "SF Pro Text", sans-serif;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: If the `font-family` is the same, and it's already set in upper CSS layers, we don't have to include it here.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+		display: flex;
+		align-items: center;
+
+		.site-preview__header-title-summary {
+			flex-grow: 1;
+			display: flex;
+			flex-direction: column;
+
+			.site-preview__header-title {
+				font-family: "SF Pro Text", sans-serif;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: It doesn't seem required

Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [730c56b](https://github.com/Automattic/wp-calypso/pull/87760/commits/730c56ba50fd0a6a39657b8060de21c647ff595f)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss (hunk 7 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [730c56b](https://github.com/Automattic/wp-calypso/pull/87760/commits/730c56ba50fd0a6a39657b8060de21c647ff595f)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss (hunk 6 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [268a610](https://github.com/Automattic/wp-calypso/pull/87760/commits/268a610acf2d3853da7fbf8a92592351b4aeae89)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss (hunk 5 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [268a610](https://github.com/Automattic/wp-calypso/pull/87760/commits/268a610acf2d3853da7fbf8a92592351b4aeae89)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx (hunk 4 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/index.tsx]: [6f8d501](https://github.com/Automattic/wp-calypso/pull/87760/commits/6f8d501ad23e3b12787727be644dd3875568fbed)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx (hunk 3 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx]: [dd13a8f](https://github.com/Automattic/wp-calypso/pull/87760/commits/dd13a8f96dc5685f01148cb227d610cf7f6c9de0)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx (hunk 2 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx]: [6f8d501](https://github.com/Automattic/wp-calypso/pull/87760/commits/6f8d501ad23e3b12787727be644dd3875568fbed)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+			flex-grow: 1;
+			display: flex;
+			flex-direction: column;
+
+			.site-preview__header-title {
+				font-style: normal;
+				font-weight: 500;
+				font-size: rem(32px);
+				line-height: 32px;
+				color: #000;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]:
```suggestion
				color: var(--studio-black);
```

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+			}
+
+			.site-preview__header-summary {
+
+				.site-preview__header-summary-link {
+					display: flex;
+					align-items: center;
+					font-size: rem(18px);
+					font-weight: 400;
+					color: #3c434a;
```
Comment (cleacos) [inline client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]:
```suggestion
					color: var(--studio-gray-70);
```

**Review by cleacos — APPROVED** [review]
LGTM!

I left two comments to use the pre-set colors instead of hardcoding them. I only reviewed the style elements for the header component.

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss (hunk 10 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [3edc54b](https://github.com/Automattic/wp-calypso/pull/87760/commits/3edc54bb0a7ceaafe2401d951a6f78a86b33c471)

File: client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss (hunk 9 above)
Comment (gogdzl) [reply client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss]: [3edc54b](https://github.com/Automattic/wp-calypso/pull/87760/commits/3edc54bb0a7ceaafe2401d951a6f78a86b33c471)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11401228

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @gogdzl, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Close Preview`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11401228).

**Pull request merged**